src/frameworks/docs/sveltekit.md

```diff
@@ -17,7 +17,7 @@ Before you get started deploying your app to Firebase,
 review the following requirements and options:
 
 - Firebase CLI version 12.1.0 or later. Make sure to
-  [install the CLI](/docs/cli#install_the_firebase_cli)
+  [install the CLI](https://github.com/firebase/firebase-tools#installation)
```

Comment:
In a similar PR from Austin I suggested this absolute link for a fix:

https://firebase.google.com/docs/cli#install_the_firebase_cli

But if there's a reason to keep everything in GH, or if only npm works for the frameworks CLI, we can use this (it needs to be fixed in a few other places I'm afraid).

Reply:
Ah, I was looking for an official page in the docs, thanks!

The one [I linked to in the change](https://github.com/firebase/firebase-tools#standalone-binary) has a command for installing "Standalone Binary" as well, but I prefer the google.com link!

I'll go ahead and look for other occurrences to replace too.

Reply:
Thanks for getting the Astro doc fixed!